tools/test.py

```diff
@@ -152,8 +152,7 @@
         else:
             # Build all tests
             if not options.build_dir:
-                print "[ERROR] You must specify a build path"
-                sys.exit(1)
+                args_error(parser, "argument --build is required")
```

Comment:
Considering that we don't have a guide on return codes of the tools, They are on shaky ground if they check return codes.

I agree that technically this is a breaking change.